cypress/integration/svg.ts

```diff
@@ -11,10 +11,10 @@ describe("SVG", () => {
                     right,
                     bottom,
                 } = rotate.getBoundingClientRect()
-                expect(Math.round(top)).to.equal(79)
-                expect(Math.round(left)).to.equal(79)
-                expect(Math.round(right)).to.equal(221)
-                expect(Math.round(bottom)).to.equal(221)
+                expect(Math.round(top)).to.equal(29)
```

Comment:
These have changed as a result of fixing doubled SVG transforms (eg outputting <rect x={20} style={{ x: 20 }} />)